### DISCUSSION TITLE
IM-FIT supports testing for Python-based ROS systems.

### DISCUSSION THREAD

**Discussion by cembglm** [inomuh/imfit#19] [Q&A] [answered]
Purpose: Provide V&V process for Python-based ROS robotic systems.

**Comment by Akerdogmus** [comment, marked as answer]
Can we use fault library for ROS systems?

**Reply by Akerdogmus** [reply]
or any robotic environment?

**Reply by cembglm** [reply]
Yes, IM-FIT has a special ROS fault library.
